src/components/Skeletons/ItemListSkeletonView.tsx

```diff
@@ -64,6 +64,7 @@ function ItemListSkeletonView({
             const opacity = gradientOpacityEnabled ? 1 - i / (numItems - 1) : 1;
             items.push(
                 <SkeletonViewContentLoader
+                    speed={3}
```

Comment:
@daledah we don't need this speed for all list loaders, it is only required for search list. Plz add a new prop `loaderSpeed` and use it.